.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: ruby
 rvm:
-  - "2.1.6"
```

Comment:
I believe the use of 2.1.6 here was intentional to ensure this project supports the version required by other internal projects.